tests/test_web/test_material_fitter.py

```diff
@@ -1,27 +1,32 @@
+import os
+
 import pytest
 
 import responses
 from tidy3d.plugins.dispersion import DispersionFitter
 
-from tidy3d.web.environment import Env
-from tidy3d.web.material_fitter import FitterOptions, MaterialFitterTask
+from tidy3d.web.core.environment import Env
+from tidy3d.web.api.material_fitter import FitterOptions, MaterialFitterTask
 
 Env.dev.active()
 
 
 @pytest.fixture
 def set_api_key(monkeypatch):
     """Set the api key."""
-    import tidy3d.web.http_management as http_module
+    import tidy3d.web.core.http_util as http_module
 
     monkeypatch.setattr(http_module, "api_key", lambda: "apikey")
 
 
 @responses.activate
 def test_material_fitter(monkeypatch, set_api_key):
-    fitter = DispersionFitter.from_file("tests/data/nk_data.csv", skiprows=1, delimiter=",")
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    parent_dir = os.path.dirname(current_dir)
+    file_path = os.path.join(parent_dir, "data/nk_data.csv")
+    fitter = DispersionFitter.from_file(file_path, skiprows=1, delimiter=",")
```

Comment:
I usually prefer to use the [`tmp_path` fixture](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) instead of creating data in-tree and having to clean it up manually later.
Also, `pathlib` is more flexible for working with paths than `os.path`.

Reply:
Good idea.